templates/default/indicators.html

```diff
@@ -1,6 +1,31 @@
-{block name=indicatorcontents}
-<a href="?m=eventlist"><img src="img/calendar.gif" alt="{$eventcount.today}"></a>&nbsp;
-<a href="?m=rtqueueview"><img src="img/24-ring-red.png" alt="{$ticketcount.critical}"></a>&nbsp;
-<a href="?m=rtqueueview"><img src="img/ring-orange.png" alt="{$ticketcount.high}"></a>&nbsp;
-<a href="?m=rtqueueview"><img src="img/ring-blue.png" alt="{$ticketcount.unread}"></a>&nbsp;
+{block name=indicators}
+
+{if $istats[eventcounter] > 1}
```

Comment:
lepiej używać notacji:
$istats.eventcounter
albo w ostateczności:
$istats['eventcounter']
ale ja bym to w ogóle nazwał:
$istats.events
Po co licznik dodawać w nazwie skoro to i tak liczba zdarzeń/wydarzeń.

Reply:
Fakt w sumie po dodaniu istats aż tak skomplikowane nazwy zmiennych nie są potrzebne. 

Chciałem uniknąć konfliktu ze zmiennymi w innych plikach. Poprawie jutro bo z telefonu webeditor słabo działa.